### DISCUSSION TITLE
We are unable to complete your signup.

### DISCUSSION THREAD

**Discussion by caiyuan3859** [vercel/community#4614] [Account Issue]
### Question

<img width="396" alt="Snipaste_2023-10-25_20-48-28" src="https://github.com/vercel/community/assets/146581397/8827f7da-9cd8-4369-a0a8-bf83ad24bcbe">
How do I solve account registration issues?

**Comment by amyegan** [comment]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps.

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Comment by caiyuan3859** [comment]
I received an email reply, but there was no clear reply. How should I solve my account registration problem?
